CHANGELOG_NEXT.md

```diff
@@ -23,6 +23,7 @@
 
 ### :house: (Internal)
 
+* refactor(sdk-metrics): remove `Gauge` and `MetricAdvice` workaround types in favor of the upstream `@opentelemetry/api` types [#5254](https://github.com/open-telemetry/opentelemetry-js/pull/5254) @chancancode
```

Comment:
ah there still needs to be an entry to `breaking` as we drop support for old API versions 🙂

Reply:
Oops, that's right, fixed!